data/src/config/config.py

```diff
@@ -33,9 +33,16 @@
 tiles_file_id_prefix: str = "vacant_properties_tiles"
 """ the prefix of the name of the tiles file generated and saved to GCP """
 
+write_production_tiles_file: bool = False
```

Comment:
Can we rename `write_production_tiles_file` to something like `should_write_production_tiles_file`?

This variable has a boolean value, but its name sounds like it could be a function. Let's make its meaning and purpose more clear.